lib/services/nodes-api-service.js

```diff
@@ -509,6 +509,26 @@ function nodeApiServiceFactory(
     };
 
     /**
+     * Remove the tag from a list of nodes
+     * @param  {String}     tagName
+     * @return {Promise}
+     */
+    NodeApiService.prototype.masterDelTagById = function(tagName) {
+        return Promise.resolve().then(function() {
+                assert.string(tagName, 'tag must be a string');
```

Comment:
@uppalk1 Be careful making that change.  An assert outside of the promise chain will generate a synchronous error and will be missed by a .catch().  Promise chains can be started with Promise.try() to handle that case.